index.html

```diff
@@ -224,7 +224,15 @@ <h2><a name="documents">Reading material</a></h2>
       </tr>
       <tr>
         <td><a target="_blank" href="http://www.iana.org/domains">IANA Domain Name Services</a></td>
-        <td>IANA maintaines and operates several key aspects of the DNS</td>
+        <td>IANA maintains and operates several key aspects of the DNS</td>
+      </tr>
+      <tr>
+        <td><a target="_blank" href="https://jpmens.net/dns">Blogged DNS links listinformation listing</a></td>
```

Comment:
'Blogged DNS links listinformation listing' looks weird, did you mean to write it like that?

Reply:
Fixed! New text -> "Blogged DNS links listing"